docs/About/Contributors.md

```diff
@@ -0,0 +1,88 @@
+# Contributors
+
+## Original Authors
+
+??? note "Chris Chafe /// ***CCRMA, Stanford University*** /// ***JackTrip Foundation***"
+
+    **2008 - **
+
+??? note  "Juan-Pablo Caceres /// ***CCRMA, Stanford University***"
+
+    **2008 - 2018**
+
+
+## Developers
+
+??? note "Aaron Wyatt /// ***Monash University***"
+
+    **2019 - **
+
+    Feature Work
+
+    * Graphical User Interface (QJackTrip)
+    * Authentication
+    * Installer Creation Scripts
```

Comment:
Can we add "IPv6 support" to my feature list?

Reply:
Done